content/posts/networkx/GSoC-2022/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "GSoC 2022"
+date: 2022-06-09
+draft: false
+description: "Getting started"
+tags: ["gsoc", "networkx"]
+displayInList: true
+author: ["kpetridis24"]
+
+resources:
+  - name: featuredImage
+    src: "GSoC_networkx_logo.png"
+    params:
+      description: "Google Summer of Code Logo with NetworkX logo"
+      showOnTop: true
+---
+
+# Intro
+I got accepted as a **GSoC** contributor, and I am so excited to spend the summer working on 
+such an incredibly interesting project. The mentors are very welcoming, communicative, fun 
+to be around, and I really look forward to collaborating with them. My application for GSoC 2022 
+can be [found here](https://summerofcode.withgoogle.com/programs/2022/projects/V1hY83XG).
+
+# About me
```

Comment:
Good update thank you. I would also add a link to your GitHub profile since it would be very relevant for GSoC.